ansible/roles/agnosticd_virtualenv/README.adoc

```diff
@@ -0,0 +1,18 @@
+= agnosticd_virtualenv
+
+Create a Python virtualenv for a host and set `ansible_python_interpreter` so that further task execution on the host will occur with this virtualenv.
+
+This is role runs from setup_runtime with a default set of Python requirements that can be set in a configs `requirements.txt`.
+
+== Configuration
+
+`agnosticd_virtualenv_requirements` - Python requirements in list format as used by `pip` module `name` argument.
+Default value loaded from lines in `configs/{{ env_type }}/requirements.txt` if it exists.
```

Comment:
`requirements.yml` is for roles

Can we make that one more explicit ? maybe `python-requirements.txt` ?

Reply:
Makes sense, changed.